sqlmesh/__init__.py

```diff
@@ -56,6 +57,8 @@ def get(cls) -> RuntimeEnv:
         except NameError:
             pass
 
+        if os.getenv("SQLMESH_PDB"):
```

Comment:
Instead of introducing a new environment variable should we consider reusing an existing [debug mode](https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/utils/__init__.py#L219-L220)?

Reply:
I thought about that, my only thought is that debug mode is common for your end users to gather logs. Do we want the simplified console output in tandem; is that fine or surprising behavior? The real beneficiary is for interactive tools like pdb, hence `SQLMESH_PDB`. Buttt I dont feel strongly. I can just use the existing debug.

Reply:
I suppose the output is actually more informative than the progress bars so it is probably fitting for SQLMESH_DEBUG in general